presto-parser/src/main/java/com/facebook/presto/sql/tree/ExistsPredicate.java

```diff
@@ -21,26 +21,26 @@
 public class ExistsPredicate
         extends Expression
 {
-    private final Query subquery;
```

Comment:
I changed this to `Node` as I assign `SymbolReference` to it. That way it is used in the same way as IN predicate where we assign `SymbolReference` to `valueList` in `ApplyNode#subqueryAssignments`